schema.sql

```diff
@@ -1,5 +1,56 @@
 /* Database schema to keep the structure of entire database. */
 
 CREATE TABLE animals (
-    name varchar(100)
+  id  INTEGER,
+  name VARCHAR(255),
+  date_of_birth DATE,
+  escape_attempts INTEGER,
+  neutered BOOLEAN,
+  weight_kg DECIMAL(10, 2)
```

Comment:
- [ ] It's very important to adhere to the requirements  in this your schema  Kindly add a new table with the name: vet, specializations,visits  all in seperately with like this colum 
- example below, 

`CREATE TABLE specializations ( 
    species_id INT, vet_id INT, 
    CONSTRAINT fk_species 
    FOREIGN KEY(species_id) 
    REFERENCES species(id), 
    CONSTRAINT fk_vets 
    FOREIGN KEY(vet_id) 
    REFERENCES vets(id)
    );`
````sql
CREATE TABLE vets (
   id int NOT NULL IDENTITY(1,1),
   name varchar(255),
   age int,
   date_of_graduation date,
   PRIMARY KEY (id)
   );
````


````sql
CREATE TABLE visits (
   animal_id int,
   vet_id int,
   visit_date date -- another name is acceptable
   );
````